smoke-test/src/test/java/org/opennms/smoketest/TopologyIT.java

```diff
@@ -879,6 +884,23 @@ public void verifyPathOutageTopologyInfo() {
     }
 
     /**
+     * This method tests that selecting an empty category does not cause a default "No focus defined" window to pop up.
+     * <p>
+     *     Temporary solution which only works if the category name is unused
+     * </p>
+     */
+    @Test(expected = NoSuchElementException.class)
+    public void verifyCollapsibleCriteriaNoDefaultFocusWindow() throws IOException, InterruptedException {
+        topologyUiPage.clearFocus();
+
+        String categoryName = "Servers";
+
+        topologyUiPage.search(categoryName).selectItemThatContains(categoryName);
+        topologyUiPage.testCase.findElementByXpath("//div[@id='gwt-uid-24']");
```

Comment:
This will fail. The uid is generated and will vary. Probably use the `topologyUiPage.getVisibleVertices()` method and search for a vertex with the label `Servers`